FreeRTOS/FreeRTOS-Plus-TCP
# Fix FreeRTOS+TCP unit tests for latest Kernel V10.6.1

Description
-----------
This PR fixes unit tests , which fail due to xMPU_SETTINGS structure defined in the latest FreeRTOS-Kernel V10.6.1
This addresses the unit tests fail issue , when a dummy xMPU_SETTINGS is defined in portmacro.h for updating the Kernel submodule pointer in this [PR ](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1013)

Test Steps
-----------
Run the unit tests locally
```
cmake -S test/unit-test -B test/unit-test/build/
make -C test/unit-test/build/ all
```

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
